packages/vm/tests/api/runBlockchain.spec.ts

```diff
@@ -101,11 +95,7 @@ tape('runBlockchain', (t) => {
 
     const vm = await setupVM({ common, genesisBlock })
 
-    st.equal(vm.blockchain.meta.genesis?.toString('hex'), testData.genesisBlockHeader.hash.slice(2))
-
     await vm.blockchain.putBlock(block)
-    const head = await vm.blockchain.getHead()
-    st.equal(head.hash().toString('hex'), testData.blocks[0].blockHeader.hash.slice(2))
```

Comment:
Same here.

Reply:
i was able to fix this by moving the test to after vm.runBlockchain() has been called.  Otherwise function returns genesis block